.github/workflows/ci.yml

```diff
@@ -10,19 +10,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        distro:
-        - centos-8
-        - centos-9
-        - fedora-35
-        - fedora-36
+        include:
+          - distro: centos-8
+            env: py36
+          - distro: centos-8
+            env: py38
+          - distro: centos-9
+            env: py39
+          - distro: fedora-35
+            env: py310
+          - distro: fedora-36
+            env: py310
```

Comment:
Nice! and it run correctly.

I wonder if include can be removed:

    matrix:
      - distro: ...
        env: ...
      ...